src/Appwrite/Platform/Workers/Certificates.php

```diff
@@ -445,30 +484,28 @@ private function updateDomainDocuments(string $certificateId, string $domain, bo
                 return;
             }
 
-            $project = $this->dbForConsole->getDocument('projects', $projectId);
+            $project = $dbForConsole->getDocument('projects', $projectId);
 
             /** Trigger Webhook */
             $ruleModel = new Rule();
-            $ruleUpdate = new Event(Event::WEBHOOK_QUEUE_NAME, Event::WEBHOOK_CLASS_NAME);
-            $ruleUpdate
+            $queueForEvents
                 ->setProject($project)
                 ->setEvent('rules.[ruleId].update')
                 ->setParam('ruleId', $rule->getId())
                 ->setPayload($rule->getArrayCopy(array_keys($ruleModel->getRules())))
                 ->trigger();
 
+
             /** Trigger Functions */
-            $ruleUpdate
-                ->setClass(Event::FUNCTIONS_CLASS_NAME)
-                ->setQueue(Event::FUNCTIONS_QUEUE_NAME)
+            $queueForFunctions
```

Comment:
$queueForFunctions does not have project, event, param, payload set.

Reply:
fixed